obal/data/playbooks/nightly/metadata.obal.yaml

```diff
@@ -3,3 +3,11 @@ help: >
 
 
   This typically uses unreleased sources taken from a git repository.
+variables:
+  nightly_sourcefiles:
+    help: Comma-separated list of absolute filepaths to be copied and used for RPM sources.
+  nightly_githash:
+    help: Git hash to be included in the package's release.
+  build_package_scratch:
+    parameter: --scratch
+    help: Set 'true' to scratch build.
```

Comment:
I feel like we should expand the metadata a bit so we can set this to a boolean type but I'm fine with starting here.

Reply:
#151 has been merged, so `--scratch` now has:

```
action: store_true
```